src/renderers/webxr/WebXRManager.js

```diff
@@ -600,6 +603,27 @@ class WebXRManager extends EventDispatcher {
 
 		};
 
+		this.createAnchor = async function ( position, rotation ) {
+
+			if ( xrFrame ) {
+
+				const anchorPose = new XRRigidTransform( position, rotation );
+				await xrFrame.createAnchor( anchorPose, this.getReferenceSpace() );
```

Comment:
Great that you added support for anchors! Are you also planning on adding support for persistent anchors? Those are more useful :-)